### PR TITLE
fixes razorwire not working and nerfs it

### PR DESCRIPTION
**Pull request by MacBlaze1**
## About The Pull Request

This PR fixes razorwire and makes it into its intended state, albeit less powerful than was originally in there. It entangles crushers for 1 second, and takes 3 RAMS to destroy, the crusher takes damage the 2 first times when ramming.
## Why It's Good For The Game

fixes are good, but this is content and complexity that was never in the game before. I believe this will strengthen fob defense while also forcing xenos to work around the razorwire, like through gas shelling and vomiting acid onto the wire or spitters using their acid spray to remove it in 2 hits. It adds more tools for engineers, while making sure crushers arent instantly breaching the FOB

## Changelog
:cl:
balance: reduced the stun on the razorwire for charging crushers from 2 seconds to half a second, this is to make sure they actually stop moving.
balance: based on input by xeno people, and testing, I have turned the damage into a flat 32 for getting entangled and unentangled. Try and use spitter acid spray or gas clouds and vomiting acid.
balance: made the entanglement last 1 second instead of 2.5 seconds.
fix: fixed razorwire being instantly destroyed by crushers.
fix: fixed razorwire not correctly entangling.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/charge_crush.dm [hunk 1]
```diff
@@ -469,11 +478,11 @@
 /obj/structure/razorwire/post_crush_act(mob/living/carbon/xenomorph/charger, datum/action/xeno_action/ready_charge/charge_datum)
 	if(!anchored)
 		return ..()
-	razorwire_tangle(charger, RAZORWIRE_ENTANGLE_DELAY * 0.5) //entangled for only half as long
+	razorwire_tangle(charger, RAZORWIRE_ENTANGLE_DELAY * 0.20) //entangled for only 20% as long or 1 second
 	charger.visible_message("<span class='danger'>The barbed wire slices into [charger]!</span>",
 	"<span class='danger'>The barbed wire slices into you!</span>", null, 5)
-	charger.Paralyze(20)
-	charger.apply_damage(rand(RAZORWIRE_BASE_DAMAGE * RAZORWIRE_MIN_DAMAGE_MULT_MED, RAZORWIRE_BASE_DAMAGE * RAZORWIRE_MAX_DAMAGE_MULT_MED), BRUTE, ran_zone(), 0, TRUE) //Armor is being ignored here.
+	charger.Paralyze(05)
```
Comment (psykzz) [inline code/modules/mob/living/carbon/xenomorph/charge_crush.dm:484]:
```suggestion
	charger.Paralyze(0.5 SECONDS)
```

File: code/modules/mob/living/carbon/xenomorph/charge_crush.dm [hunk 2]
```diff
@@ -391,11 +391,20 @@
...
 	for(var/m in buckled_mobs)
 		unbuckle_mob(m)
 	return (CHARGE_SPEED(charge_datum) * 20) //Damage to inflict.
 
+/obj/structure/razorwire/pre_crush_act(mob/living/carbon/xenomorph/charger, datum/action/xeno_action/ready_charge/charge_datum)
+	if(CHECK_BITFIELD(resistance_flags, INDESTRUCTIBLE) || charger.is_charging < CHARGE_ON)
+		charge_datum.do_stop_momentum()
+		return PRECRUSH_STOPPED
+	if(anchored)
+		. = (CHARGE_SPEED(charge_datum) * 50) //Damage to inflict. 3 rams, 2 times taking damage.
```
Comment (psykzz) [inline code/modules/mob/living/carbon/xenomorph/charge_crush.dm:404]: >//Damage to inflict. 3 rams, 2 times taking damage.

This comment doesn't make much sense.
What does 3 rams mean here?

Comment (MacBlaze1) [reply code/modules/mob/living/carbon/xenomorph/charge_crush.dm:404]: 3 charges against the razorwire, actually charging it as a crusher, you take damage the 2 first times, and then on the third it just is broken.

Comment (psykzz) [reply code/modules/mob/living/carbon/xenomorph/charge_crush.dm:404]: Ah i see what you mean now

**Pull request merged**